VTFLib/VTFFile.cpp

```diff
@@ -3031,7 +3032,7 @@ static SVTFImageFormatInfo VTFImageFormatInfo[] =
 	{},
 	{},
 	{},
-	{},
+	{ "BC6H",					8,  0,  0,  0,  0,  0, vlTrue,  vlTrue  },			// IMAGE_FORMAT_BC6H
```

Comment:
This will need to get moved to after IMAGE_FORMAT_BC7 when you update the enum (see the later comment)

Reply:
✔